bin/update-yarn

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# inspired by https://gist.github.com/maxrimue/16e272c4e0b8a8ac9dd5570e180e08c0
+
+# this script is only for CircleCI
+$CIRCLECI || exit 0
+
+if [[ $CIRCLE_BRANCH != *"greenkeeper"* ]]; then
+	# Not a GreenKeeper Pull Request, aborting
+	exit 0
+fi
+
+cd $CIRCLE_PROJECT_REPONAME
```

Comment:
I am worried this may need to be `cd $HOME/$CIRCLE_PROJECT_REPONAME` but I wanted to see a run first.